sensorimotor/sensorimotor/sensorimotor_experiment_runner.py

```diff
@@ -149,8 +151,9 @@ def feedLayers(self, sequences, tmLearn=True, tpLearn=None, verbosity=0,
      sensorimotorSequence,
      sequenceLabels) = sequences
 
-    self.tm.mmClearHistory()
-    self.tp.mmClearHistory()
+    if clearHistory:
```

Comment:
Remove `clearHistory` param and move following two lines to `runTest` in `experiment.py`.